core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.hpp

```diff
@@ -733,13 +734,22 @@ class OpenMPTargetExecTeamMember {
   using space = execution_space::scratch_memory_space;
 
  public:
+  // FIXME: OPENMPTARGET - 8 bytes at the begining of the scratch space for each
+  // league is saved for reduction. It should actually be based on the ValueType
+  // of the reduction variable.
   inline OpenMPTargetExecTeamMember(
       const int league_rank, const int league_size, const int team_size,
       const int vector_length  // const TeamPolicyInternal< OpenMPTarget,
                                // Properties ...> & team
       ,
       void* const glb_scratch, const int shmem_size_L1, const int shmem_size_L2)
-      : m_team_shared(nullptr, 0),
+      : m_team_shared(((char*)glb_scratch +
+                       league_rank * (shmem_size_L1 + shmem_size_L2 + 8)),
```

Comment:
Should the pointer be shifted by another 8 bytes since you said you are saving it for reductions?

Reply:
Ah yes the 8 was supposed to be for 8 bytes. I forgot in the implementation it should be 64. I will fix it. 

> Did you mean to comment out code in the incremental tests?

I am not commenting any code in the incremental tests. I am only protecting an input size for the OpenMPTarget backend.

Reply:
You are commenting a lot of code in `core/unit_test/incremental/Test12a_ThreadScratch.hpp`

Reply:
Yes sorry about that. I did not commit that file 🙄 . Also made a small change in the same tests, where I replace the hard coded scratch level to a const variable.